torch/_inductor/codegen/triton.py

```diff
@@ -1814,9 +1814,31 @@ def codegen_kernel(self, name=None):
 
         code = IndentedBuffer()
 
-        size_hints = [
-            next_power_of_2(V.graph.sizevars.size_hint(numel)) for numel in self.numels
-        ]
+        size_hints = []
+        for numel in self.numels:
+            numel_hint = V.graph.sizevars.symbolic_hint(numel)
+            if not isinstance(numel_hint, int):
+                # This default heuristic hint was picked carefuly:
+                #
+                #  - It is NOT a multiple of 16 (which would trigger
+                #    a multiple_of_16 hint), since in general we can't
+                #    guarantee that unbacked symints are this
```

Comment:
I don't think this applies here since this hint controls triton block sizes which are always powers of two, and the hint was always passed to `next_power_of_2` anyway.

Reply:
Ah, you're right